document/js-api/index.bs

```diff
@@ -730,8 +730,14 @@ which can be simultaneously referenced by multiple {{Instance}} objects. Each
 <h3 id="globals">Globals</h3>
 
 <pre class="idl">
+enum ValtypeEnum {
```

Comment:
@littledan I'm wondering if we should either hoist it in the spec to allow it to be re-used, but there's no use-case yet. Or I would like to scope it here. Does an inline enum in the GlobalDescriptor struct work or should I change its name?

Reply:
WebIDL doesn't really have scoping; you definitely can't put the enum inside the dictionary. If you want to avoid overlaps, you could prefix the name with WebAssembly.

I'd suggest avoiding using an abbreviated name here. Also, I don't think you need to include Enum in the name of an enum.

Putting that all together, you could use WebAssemblyValueType as the name.

Reply:
Ok, sounds good to me; thanks.